scripts/scripts.js

```diff
@@ -43,13 +43,35 @@ export function addLdJsonScript(parent, json) {
   parent.append(script);
 }
 
+/** allow for link attributes to be added into link text
+ * ex: Link Text{target=blank|rel=noopener}
+ * @param main
+ */
+export function buildLinks(main) {
+  main.querySelectorAll('a').forEach((a) => {
+    const match = a.textContent.match(/(.*){([^}]*)}/);
+    if (match) {
+      // eslint-disable-next-line no-unused-vars
+      const [_, linkText, attrs] = match;
+      a.textContent = linkText;
+      // match all attributes between curly braces
+      attrs.split('|').forEach((attr) => {
+        const [key, ...values] = attr.split('=');
+        const value = values.join('=');
+        //  a.setAttribute(key.trim(), value.trim());
+        a.setAttribute(key, value);
+      });
+    }
+  });
+}
+
 /**
  * Decorates paragraphs containing a single link as buttons.
  * @param {Element} element container element
  */
 export function decorateButtons(element) {
   element.querySelectorAll('a').forEach((a) => {
-    a.title = a.title || a.textContent;
+    if (a.title) a.title = a.title || a.textContent;
```

Comment:
tested on all other links? I feel this will prevent title to be populated on any link.

Reply:
this line doesn't really do anything - good catch!